_clanky/znalostní-grafy-02-rdf.md

```diff
@@ -0,0 +1,339 @@
+
+~~~~~~
+<https://data.gov.cz/zdroj/ovm/00025593>   <http://www.w3.org/1999/02/22-rdf-syntax-ns#type>   <https://schema.org/Organization>, <http://data.europa.eu/m8g/PublicOrganisation> ;
+                                           <https://schema.org/name>   "Český statistický úřad" ;
+                                           <https://data.gov.cz/slovník/ovm/datováSchránka>   <https://data.gov.cz/zdroj/datové-schránky/2gfaasy> ;
+                                           <https://data.gov.cz/slovník/ovm/právníForma>   <https://data.gov.cz/zdroj/ovm/právní-forma/325> .
+<https://data.gov.cz/zdroj/datové-sady/http---vdb.czso.cz-pll-eweb-package_show-id-290038r19>   <http://purl.org/dc/terms/publisher>   <https://data.gov.cz/zdroj/ovm/00025593> .
+~~~~~~~~~~~~
+
+Dále je možné využít *prefixování*, abychom nemuseli v IRI vypisovat jejich opakující se části. Prefix umožňuje zkrátit všechna IRI se stejným začátkem tak, že místo opakujícího se začátku uvedeme identifikátor prefixu. Prefix je deklarován přímo jako součást zápisu RDF trojic. Následující příkad zavedení a využití prefixů demonstruje. Jedná se o zápis stejných dat, jako v předchozím příkladu.
+
+~~~~~~
+@prefix data: <https://data.gov.cz/zdroj/>
+@prefix schema: <https://schema.org/>
+@prefix isa: <http://data.europa.eu/m8g/>
+@prefix ovm: <https://data.gov.cz/slovník/ovm/>
+@prefix dct: <http://purl.org/dc/terms/>
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+data:ovm/00025593   rdf:type   schema:Organization, isa:PublicOrganisation ;
+                    schema:name   "Český statistický úřad" ;
+                    ovm:datováSchránka   data:datové-schránky/2gfaasy ;
+                    ovm:právníForma   data:právní-forma/325 .
+data:datové-sady/http---vdb.czso.cz-pll-eweb-package_show-id-290038r19   dct:publisher   data:ovm/00025593 .
+~~~~~~~~~~~~
+
+Zkrátit zápis můžeme také vyjádřením predikátu `http://www.w3.org/1999/02/22-rdf-syntax-ns#type` symbolem `a`, jak ukazuje následující zápis.
+
+~~~~~~
+@prefix data: <https://data.gov.cz/zdroj/>
+@prefix schema: <https://schema.org/>
+@prefix isa: <http://data.europa.eu/m8g/>
+@prefix ovm: <https://data.gov.cz/slovník/ovm/>
+@prefix dct: <http://purl.org/dc/terms/>
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+data:ovm/00025593   a   schema:Organization, isa:PublicOrganisation ;
+                    schema:name   "Český statistický úřad" ;
+                    ovm:datováSchránka   data:datové-schránky/2gfaasy ;
```

Comment:
Tohle kvůli lomítku asi nebude validní. Otázka je, jestli to tu vadí. Znamenalo by to přidat další prefixy.

Reply:
Zavedl jsem prefixy, už je to validní